proto/stride/ratelimit/tx.proto

```diff
@@ -5,7 +5,7 @@ option go_package = "github.com/Stride-Labs/stride/v4/x/ratelimit/types";
 
 // Msg defines the Msg service.
 service Msg {
-  rpc AddQuota(MsgAddQuota) returns (MsgAddQuotaResponse);
+  rpc SetQuota(MsgSetQuota) returns (MsgSetQuotaResponse);
```

Comment:
so sorry for misleading you. The Message should stay as `MsgAddQuota` - I just mean the keeper function can be changed to `SetQuota`

So in the message implementation, it would do something like:

```
func MsgAddQutoa(quota) {
    _, found := k.GetQuota(quota.Name)
    if found {
       error already exists
    }
    k.SetQuota(quota)
}

```